backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc

```diff
@@ -10,23 +10,22 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You install the Openshift API for Data Protection (OADP) with Amazon Web Services (AWS) by installing the OADP Operator, configuring AWS for Velero, and then installing the Data Protection Application.
+You install the Openshift API for Data Protection (OADP) with Amazon Web Services (AWS) by installing the OADP  Operator. The Operator installs link:https://{velero-domain}/docs/v1.7/[Velero 1.7].
```

Comment:
two spaces between OADP and Operator (does it matter in adoc?)

Reply:
No, it doesn't matter in Asciidoc but I will fix it. Thanks!